elements/pfe-sass/mixins/_custom-properties.scss

```
@@ -0,0 +1,172 @@
+// New broadcast mixin, for use with content components
+//  1st arg:  lookup surface value from theme
+//  2nd arg:  list of broadcast variable - optional
+//  3rd arg:  identify the color set, default is surface - optional
+//
+//  usage:
```

so they can go look it up and see it in action
```suggestion
//  container usage, i.e. pfe-card:
```